lib/mysqlDaoQueryHelper.js

```diff
@@ -261,73 +262,103 @@ class MysqlDaoQueryHelper {
 
     _deleteBase(whereClause) {
         let sqlObject =
-            Squel.delete({parameterCharacter: '!!@@##$$%%'}) // See comment in _getBase for explanation of paramChar
+            this._squel.delete({parameterCharacter: '!!@@##$$%%'}) // See comment in _getBase for explanation of paramChar
                 .from(this._tableName);
 
         sqlObject = this._appendWhereClause(sqlObject, whereClause);
         return sqlObject;
     }
 
     /**
-     * @param {object} obj
-     * @param {object} [config]
-     * @param {boolean} [config.dontCleanMysqlFunctions] - defaults to true; includes CURRENT_TIMESTAMP and NOW();
-     *
-     * @returns {object} - cleans the values and returns an object with snake case props --> clean values
+     * @private
+     * @param {Object[]} objs
+     * @param {Boolean} [dontCleanMysqlFunctions] whether or not to allow MySQL functions to exist in values
+     * @returns {Object[]} - cleans the list of values and returns objects with snake_case props --> clean values
      */
-    _cleanAndMapValues(obj, config) {
-        const defaultConfig = {
-            dontCleanMysqlFunctions: true
-        };
+    _cleanAndMapValueList(objs , dontCleanMysqlFunctions = true) {
+        // Generate the snake-case property mappings for the first row only
+        const propertyMappings = MysqlDaoQueryHelper._getPropertyMappings(objs[0]);
+        return objs.map(obj => this.__cleanAndMapValues(obj, propertyMappings, dontCleanMysqlFunctions));
+    }
 
-        config = _.merge(defaultConfig, config);
+    /**
+     * @private
+     * @param {Object} obj
+     * @param {Boolean} [dontCleanMysqlFunctions] whether or not to allow MySQL functions to exist in values
+     * @returns {Object} - cleans the values and returns an object with snake_case props --> clean values
+     */
+    _cleanAndMapValues(obj, dontCleanMysqlFunctions = true) {
```

Comment:
Minor: I found the three function names sort of hard to understand initially (`_cleanAndMapValueList`, `_cleanAndMapValues`, `__cleanAndMapValues`). 

I'd rename the double-underscore function to just be a little more specific (like _cleanAndMapValuesUsingPropertyMap or something), get rid of the `_cleanAndMapValueList` method and just make this method more defensive. You could type the first param as an object or array  `{Object|Object[]}`, and then have an if-statement:
```javascript
if (Array.isArray(objs)) {
    const propertyMap = MysqlDaoQueryHelper._getPropertyMappings(objs[0]);
    objs.map(obj => this._cleanAndMapValuesWithPropertyMap(obj, propertyMap, dontCleanMysqlFunctions));
} else {
    const propertyMap = MysqlDaoQueryHelper._getPropertyMappings(obj);
    this._cleanAndMapValuesWithPropertyMap(objs, propertyMap, dontCleanMysqlFunctions);
}
```